internal/metrics/metricsManager.go

```diff
@@ -475,12 +477,12 @@ func (m *MetricsManager) FileSystemUsage(usageInPercentage float64) {
 	m.serviceLogicFilesystemUsagePercentage.WithLabelValues().Set(usageInPercentage)
 }
 
-func (m *MetricsManager) MonitoredHostsCount(monitoredHosts int64) {
-	m.serviceLogicMonitoredHosts.WithLabelValues(hosts).Set(float64(monitoredHosts))
+func (m *MetricsManager) MonitoredHostsDurationMs(monitoredHostsMs float64) {
+	m.serviceLogicMonitoredHostsDurationMs.WithLabelValues().Observe(monitoredHostsMs)
```

Comment:
`WithLabelValues` seems to follow a fluent interface pattern. I think it can be dropped altogether if called with no arguments.